src/sentry/snuba/metrics_performance.py

```diff
@@ -442,11 +443,30 @@ def top_events_timeseries(
                 if zerofill_results
                 else item["data"],
                 "order": item["order"],
+                # XXX: How do I know if this is right?
+                "meta": {"isMetricsExtractedData": top_events_builder.use_on_demand},
```

Comment:
I include the builder's use_on_demand property, however, I don't know if each specific Snuba result is actually using on-demand metrics or not.